### ISSUE TITLE
Vite 2.9

### ISSUE THREAD

**Issue by jenky** [lhapaipai/vite-plugin-symfony#6]
```
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: vite-plugin-symfony@0.3.2
npm WARN Found: vite@2.9.14
npm WARN node_modules/vite
npm WARN   dev vite@"^2.9.9" from the root project
npm WARN   2 more (@vitejs/plugin-vue, unplugin)
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer vite@"~2.8" from vite-plugin-symfony@0.3.2
npm WARN node_modules/vite-plugin-symfony
npm WARN   dev vite-plugin-symfony@"^0.3.0" from the root project
```

**Comment by lhapaipai** [comment]
can you read the #1 please ?

**Comment by jenky** [comment]
I believe it is developer's responsibility to ensure the build successful. You can update readme about the caveat or even tag a new major version to address the issue. That's the purpose of semver. By locking a dependency at specific version, you are throwing away new features, performance improvements, bug fixes and security patches, etc.. which I think it is not a good practice.

**Comment by lhapaipai** [comment]
hi @jenky , 
I agree with you, I will change that in the next update.
But please, **also** consider it a good practice to put more than one error message when you open an issue.That doesn't encourage me to read you or reply to you.
good afternoon

**Comment by jenky** [comment]
Thank you. I'll put that in mind. Cheers!

**Comment by lhapaipai** [comment]
the issue is solved with `v0.4.0` release.
